test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml

```diff
@@ -13,7 +13,6 @@ spec:
       labels:
         app: csi-gce-pd-driver
     spec:
-      serviceAccount: csi-gce-pd
```

Comment:
don't we still need to specify a service account?

Reply:
it's always going to be specified by a generated service account. Removing this dummy default value causes it to fail faster if not specified

Reply:
You mean every pod automatically gets a service account injected into it?